handbook/design/index.md

```diff
@@ -10,6 +10,22 @@ Here are a few of the things we strive to make true:
 - Communicate openly, and solicit feedback frequently
 - To be egoless when soliciting responses to our work
 
+## How to work with design
+
+*This section is WIP*
+
+Advice on how to best work with the design team at Sourcegraph. 
+
+We wanted to document this as design is new to Sourcegraph, and it might not be immediately clear how to get the most out of the design team. 
+
+- If you're in a pinch, the design team is here to help, so don't hesitate to solicit feedback from us. Even if you think your problem might not pertain to design, don't leave it to chance
+- Generally, we favour an asynchronous approach to communication, this gives us some time to validate our responses
+- An example of a time to use immediate communication would be: if you're working from a design file, and you feel that something is missing
```

Comment:
What are you trying to communicate with these two points? As a team we strive to communicate asynchronously, so how is this different from that?